app/src/test/java/com/example/concordia_campus_guide/Activities/PathsViewModelTest.java

```diff
@@ -59,4 +67,80 @@ public void arePlacesSeparatedByATunnelTest() {
     public void areInSameBuildingTest() {
         assert (pathsViewModel.areInSameBuilding(fromRoom, toRoom));
     }
+
+    @Test
+    public void deg2rad() {
+        assertEquals(0,pathsViewModel.deg2rad(0),0.001);
+        assertEquals(0.017453292519943295,pathsViewModel.deg2rad(1),0.001);
+        assertEquals(Math.PI,pathsViewModel.deg2rad(180),0.001);
+    }
+
+    @Test
+    public void getDistanceFromLatLonInKm() {
+        assertEquals(0.43287270110863557, pathsViewModel.getDistanceFromLatLonInKm(45.4972685,-73.5789475,45.493622,-73.577003 ), 0.001);
+    }
+
+    @Test
+    public void adaptOutdoorDirectionsToInfoCardList() {
```

Comment:
Put the word "Test" at the end of the method title. Applicable for all the ones below as well